### DISCUSSION TITLE
Use signed routes instead of auth scafolding

### DISCUSSION THREAD

**Discussion by StaffCollab** [filamentphp/filament#7733] [Help] [answered]
I have a project that has an account table, and each account has an email.

I prefer instead of asking each customer to create a login and user profile and associate it with their account, I rather want to be able to send emails with signed routes to the customer to enable them to get into their account without logging in.
Is there a way in filament to do so easily?

Worth mentioning that this is for a second panel, the first panel is using regular auth and user table.

**Comment by danharrin** [comment, marked as answer]
You can create a signed route that logs the user into their account and redirects them to Filament, we don't need to do anything
